nav2_system_tests/maps/empty.yaml

```diff
@@ -0,0 +1 @@
+/home/melih/ros2_all_ws/nav2_dashing_ws/src/navigation2/nav2_bringup/launch/turtlebot3_world.yaml
```

Comment:
Local path, this should be relative

Also `empty` appears to have a non-empty box in it. Is the intention to navigate inside the fixed box? If so why so much border? If its both, `empty` isn't an accurate name

Reply:
> Only changes are just in the path, maybe also the name but up to you.

This map is for the empty room gazebo world. I will change the name to empty_room because it is 10x10 meter room with nothing in it.

> On a less-blocking perspective: what's the goal of this? You mention RL, but there's no RL tests to go along with this, training weights, and RL is still in testing. Does this make more sense targeted to the experimental branch?

@mhpanah will use them in his branch. I use them to test Navigation2 with dynamic obstacles. 

> Given that these are all grouped together for a usecase, I might add a `rl_tests` folder and put it all in here to keep it all together

RL is only one example. They can be used for testing planners such as TEB and DWB. If I am not wrong, TEB should work in an environment with dynamic obstacles. I tested Nav2 using these models and DWB works if the dynamic obstacles are slower than the robot. Also, I noticed that recovery behaviors such as spinning cause issues in an environment with dynamic obstacles.